index.js

```diff
@@ -1,301 +1,314 @@
-    .option('-f, --logfile <file>', 'adds log files to config', utils.appender(), [])
-    .option('-e, --exclude <file>', 'exclude files from logdir', utils.appender(), [])
-    .option('-r, --exclude-regex <pattern>', 'filter out lines matching pattern')
-    .option('-n, --hostname <hostname>', `uses alternate hostname (default: ${os.hostname().replace('.ec2.internal', '')})`)
-    .option('-t, --tags <tags>', 'add tags for this host, separate multiple tags by comma', utils.appender(), [])
-    .option('-l, --list [params]', 'show the saved configuration (all unless params specified)', utils.appender(), false)
-    .option('-u, --unset <params>', 'clear some saved configurations (use "all" to unset all except key)', utils.appender(), [])
-    .option('-w, --winevent <winevent>', 'set Windows Event Log Names (only on Windows)', utils.appender(), [])
-    .option('-s, --set [key=value]', 'set config variables', utils.appender(), false)
-    .on('--help', () => {
-        console.log('  Examples:');
-        console.log();
-        console.log('    $ logdna-agent --key YOUR_INGESTION_KEY');
-        console.log('    $ logdna-agent -d /home/ec2-user/logs');
-        console.log('    $ logdna-agent -d /home/ec2-user/logs -d /path/to/another/log_dir  # multiple logdirs in 1 go');
-        console.log('    $ logdna-agent -d "/var/log/*.txt"                                 # supports glob patterns');
-        console.log('    $ logdna-agent -d "/var/log/**/myapp.log"                          # myapp.log in any subfolder');
-        console.log('    $ logdna-agent -f /usr/local/nginx/logs/access.log');
-        console.log('    $ logdna-agent -f /usr/local/nginx/logs/access.log -f /usr/local/nginx/logs/error.log');
-        console.log('    $ logdna-agent -t production                                       # tags');
-        console.log('    $ logdna-agent -t staging,2ndtag');
-        console.log('    $ logdna-agent -w System                                           # Windows System event logs (all providers)');
-        console.log('    $ logdna-agent -l tags,key,logdir                                  # show specific config parameters');
-        console.log('    $ logdna-agent -l                                                  # show all');
-        console.log('    $ logdna-agent -u tags,logdir                                      # unset specific entries from config');
-        console.log('    $ logdna-agent -u all                                              # unset all except LogDNA API Key');
-        console.log();
-    })
-    .parse(process.argv);
-
-if ((os.platform() === 'win32' && require('is-administrator')()) || process.getuid() <= 0) {
-    let conf_file = program.config || config.DEFAULT_CONF_FILE;
-    debug(`Path to Configuration File: ${conf_file}`);
+var config = require('./lib/config')
+var processed
+
+commander.title = 'logdna-agent'
```

Comment:
This should remain as `process.title = 'logdna-agent'`

Reply:
Link to docs: https://nodejs.org/api/process.html#process_process_title

Reply:
`process` is now relegated to the added function for processing command line args versus direct calling it with a test.  `commander.title` is still the `Commander` instance, so I'm not sure what you mean here?

Reply:
Fixed per the docs.  Thanks!